2020-10-14 10:25:16,311 - trace.py[line:122] - INFO: Task cc.add[976bff9f-2b31-4f4f-82fb-d23e157e2f0f] succeeded in 6.016988113988191s: 4
